specification/synapse/data-plane/Microsoft.Synapse/preview/2022-08-01-preview/linkConnections.json

```diff
@@ -790,6 +858,19 @@
         }
       }
     },
+    "LinkConnectionRefreshStatus": {
+      "type": "object",
+      "properties": {
+        "refreshStatus": {
+          "type": "string",
+          "description": "Link connection refresh status"
+        },
```

Comment:
Should this be enum with modelAsString=true ?

Reply:
And use "readonly" if properties is only used in response and cannot written by customer.

Reply:
Because the value of refreshStatus can be changed, it's better to use string type for now. 

readOnly: true is added.